packages/sonarwhal/src/lib/cli/wizards/templates/new-rule/readme.md.hbs

```diff
@@ -1,24 +1,52 @@
-# {{name}} (`{{normalizedName}}`)
+# {{name}} (`{{packageName}}`)
 
 {{description}}
 
 ## Why is this important?
 
-Explain why this {{#if multi}}package{{else}}rule{{/if}} is important for your users
+Explain why this {{#if isMulti}}package{{else}}rule{{/if}} is important for your users
 
-## What does the {{#if multi}}package{{else}}rule{{/if}} check?
+## How to use {{#if isMulti}}these rules{{else}}this rule{{/if}}?
```

Comment:
I was thinking that maybe we should move this after `## Further Reading`?

Reply:
I'm ok, but we will have to update all packages. I've opened https://github.com/sonarwhal/sonarwhal/issues/1018 to do this.